app/code/community/Bolt/Boltpay/Helper/Data.php

```diff
@@ -32,6 +32,15 @@
  */
 class Bolt_Boltpay_Helper_Data extends Mage_Core_Helper_Abstract
 {
+    /**
+     * @var string The Bolt sandbox url for the javascript
+     */
+    const JS_URL_TEST = 'https://connect-sandbox.bolt.com';
```

Comment:
I'm pretty sure this is already defined somewhere.

Reply:
Yeah it was already defined in Checkout_Boltpay block. I plan to refactor Block in the separate PR.